### PR TITLE
Personal item: Old space lizard plushie

### PR DESCRIPTION
**Pull request by nkokic**
## About the PR
Added a personal item for 'Darting Groth' for Stedar

## Why / Balance
Personal Item.

## Technical details
Added new YML code using the space lizard plushie item.
Plushie is wearable as helmet clothing and has inhand sprites.

## Media
imageimageimage
imageimage

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame, or this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**

🆑
- add: Added Darting Groth's old space lizard plushie. Feels very lucky.
`

### CODE REVIEW

**Comment by nkokic** [comment]
Copyright in meta.json in rsi might be incorrect.
Not sure what to link or how to find appropriate one.

File: Resources/Textures/_Umbra/Objects/Fun/Toys/plushie_old_space_lizard.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Dart111 modified OldPlushieSpaceLizard icon from PlushieSpaceLizard, head and inhand sprites made by Nikovnik (github: nkokic)",
```
Comment (TsjipTsjip) [inline Resources/Textures/_Umbra/Objects/Fun/Toys/plushie_old_space_lizard.rsi/meta.json:4]: Needs a commit reference for the purposes of source permanence.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Looks good, GM approval for the item is given. Can be merged when tested in game, verified to be working and the below comment is addressed.

File: Resources/Textures/_Umbra/Objects/Fun/Toys/plushie_old_space_lizard.rsi/meta.json (hunk 1 above)
Comment (nkokic) [reply Resources/Textures/_Umbra/Objects/Fun/Toys/plushie_old_space_lizard.rsi/meta.json]: Found out hot to search around the repository and to do "git blame".
Modified it and I think it should be fine now.

**Review by TsjipTsjip — APPROVED** [review]

**Pull request merged**